apt/make_debs.go

```diff
@@ -70,14 +111,20 @@ func part2(base, product, extra string) {
 		OrPanic(os.MkdirAll(base, 0755))
 		OrPanic(copyFile("curio", path.Join(base, "curio")))
 		OrPanic(copyFile("sptool", path.Join(base, "sptool")))
+		base = path.Join(dir, "etc", "systemd", "system")
```

Comment:
This part is used for user defined custom services. We should use the system default /lib/systemd/system/